public_html/wp-content/plugins/wc-post-types/inc/rest-api.php

```diff
@@ -44,6 +44,20 @@ function register_sponsor_post_meta() {
 			'auth_callback' => __NAMESPACE__ . '\meta_auth_callback',
 		)
 	);
+	register_post_meta(
+		'wcb_sponsor',
+		'_wcpt_sponsor_first_time',
```

Comment:
@iandunn I'm wondering what's `wcb`? 
For prefix of `_first_time` for other CPT, I'm all using `wcb`, but for sponsor, I'm using `wcpt` and is due to this [line](https://github.com/WordPress/wordcamp.org/blob/production/public_html/wp-content/plugins/wc-post-types/wc-post-types.php#L1392). If `wcb` is considered a necessary naming here, I guess it needs customizing for `first_time`

Reply:
I think it's a legacy from when the `wc-post-type` code was in the old `wordcamp-base` theme. It's a bad practice to put code inside a theme if it's not related to the appearance of a site, because then post types will be unregistered etc if you switch themes. Eventually the code was extracted into a plugin, but the original `wcb` prefix was kept for convenience.

🤔 I'm not sure why the `wc-post-types` plugin uses `wcpt` to prefix sponsor posts, that seems wrong to me. I'm guessing it was just an oversight. I think you were right to keep things consistent 👍🏻 

Kudos for noticing details like that and wanting to understand the background 👍🏻

Reply:
Updated in 3f85cc7517eac6897539c4d6b24676ad3a34ecb2